docs/source/en/perf_train_gpu_many.md

```diff
@@ -199,165 +222,213 @@ La | Lb | Lc
 a2 | b2 | c2
 ```
 
-In a way this is the same horizontal slicing, as tensor parallelism, if you imagine the typical DNN diagram. Vertical slicing is where one puts whole layer-groups on different GPUs. But it's just the starting point.
+In a way, this is the same horizontal slicing as tensor parallelism, as opposed to Vertical 
+slicing, where one puts whole layer-groups on different GPUs. Now let's see how this works: 
+
+Each of these GPUs will get the usual mini-batch as it works in DP:
 
-Now each of these GPUs will get the usual mini-batch as it works in DP:
 ```
 x0 => GPU0
 x1 => GPU1
 x2 => GPU2
 ```
 
-The inputs are unmodified - they think they are going to be processed by the normal model.
-
-First, the inputs hit the layer La.
-
-Let's focus just on GPU0: x0 needs a0, a1, a2 params to do its forward path, but GPU0 has only a0 - it gets sent a1 from GPU1 and a2 from GPU2, bringing all pieces of the model together.
+The inputs are passed without modifications as if they would be processed by the original model.
 
-In parallel, GPU1 gets mini-batch x1 and it only has a1, but needs a0 and a2 params, so it gets those from GPU0 and GPU2.
+First, the inputs get to the layer La. What happens at this point?
```

Comment:
Not sure if there is a typo here?

Reply:
If you're talking about "La", that's the name of a layer in the example:
```
La | Lb | Lc
---|----|---
a0 | b0 | c0
a1 | b1 | c1
a2 | b2 | c2
```

Reply:
Ok thanks!